client/log.go

```diff
@@ -26,6 +26,11 @@ func NewLog(unicode bool) *Log {
 	return client
 }
 
+// WithLabels adds labels map to metrics
+func (c *Log) WithLabels(labels map[string]string) Client {
```

Comment:
I would rename this to `(l *Log)`

Reply:
We use same var names in all that package as `c` - means client here